src/jlfrontend.scm

```diff
@@ -131,10 +131,15 @@
           (block
            ,loc
            (call (core eval) ,name ,x)))
-       (= (call eval m ,x)
-          (block
-           ,loc
-           (call (core eval) m ,x)))
+       (if (&& (call (top isdefined) (core Main) (quote Base))
+               (call (top isdefined) (top Base) (quote @deprecate)))
+           (call eval
+                 (quote
+                  (macrocall (top @deprecate)
```

Comment:
I'm worried this'll get missed in the deprecation removal. Add a note in deprecated.jl to remove this?